test/test-sps.js

```diff
@@ -0,0 +1,25 @@
+window.testSPS = new Uint8Array([
```

Comment:
Given these are pretty tiny, I would consider just inlining them into the test case so you don't have go searching to find them. Not worth holding up the PR for, though.